README.md

```diff
@@ -301,6 +313,11 @@ not return `true`, the resource will not end up in the final collection that is
 
 Required for HTTP method: GET with query string
 
+**delete() (optional)**
+
+This function will be called when the resource is deleted from the collection. You may use this to clean up or disable things tied to your resource instance.
+*It can not throw any exceptions!*
```

Comment:
This is actually a problem. It should be allowed to, in which case deletion should be cancelled. That is also how we treat construction and editing, and it could be a useful hook for people to protect certain resources from deletion.